test/integration/imagerepository_test.go

```diff
@@ -55,6 +63,14 @@ func TestImageRepositoryCreate(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 
+	clusterAdminKubeClient, err := testutil.GetClusterAdminKubeClient(clusterAdminKubeConfig)
```

Comment:
Helper function marked as "do not use, use create project instead" that takes the clusterAdminKubeConfig.  This is tedious to read, it had to be tedious to write.